engine/gui/src/test/test_gui_clipping.cpp

```diff
@@ -774,7 +774,7 @@ TEST_F(dmGuiClippingTest, TestRenderOrder_ClipperStraight) {
  *   - b (layer1)
  * - c
  *
- * Expected order: a, b, c
+ * Expected order: a, c, b
```

Comment:
I guess I wouldn't have expected so many cases to change.
Are we expecting a big change for the users?